src/soundsourceproxy.cpp

```diff
@@ -219,24 +170,18 @@ void SoundSourceProxy::loadPlugins() {
     s_soundSourceProviders.registerProvider(Mixxx::SoundSourceProviderPointer(
             new Mixxx::SoundSourceProviderCoreAudio));
 #endif
-#ifdef __FFMPEGFILE__
-    // FFmpeg currently overrides all other built-in providers
-    // if enabled
-    s_soundSourceProviders.registerProvider(Mixxx::SoundSourceProviderPointer(
-            new Mixxx::SoundSourceProviderFFmpeg));
-#endif
 
     // Scan for and initialize all plugins.
     // Loaded plugins will replace any built-in providers
     // that have been registered before (see above)!
     const QList<QDir> pluginDirs(getSoundSourcePluginDirectories());
-    foreach (QDir dir, pluginDirs) {
-        qDebug() << "Loading SoundSource plugins" << dir.path();
-        const QStringList files(dir.entryList(
+    for (const auto& pluginDir: pluginDirs) {
```

Comment:
The plugins are registered with there priority hint. This requires that the priority hint is set cooperative.
Can we guarantee this or do we need a facility to set the priority from the Mixxx source.
Or should the priority be read from a config file? Any ideas?